src/app/Sidebar/RightSidebar.js

```diff
@@ -18,41 +18,24 @@ import SignUp from '../../components/Sidebar/SignUp';
   state => ({
     authenticated: getIsAuthenticated(state),
     authenticatedUser: getAuthenticatedUser(state),
-    followingList: getFollowingList(state),
-  }),
+    recommendations: getRecommendations(state),
+  }), {
+    updateRecommendations,
+  },
 )
 export default class RightSidebar extends React.Component {
   static propTypes = {
     authenticated: PropTypes.bool.isRequired,
     authenticatedUser: PropTypes.shape().isRequired,
-    followingList: PropTypes.arrayOf(PropTypes.string).isRequired,
+    recommendations: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
Can we use `PropTypes.shape()` instead of object with correct properties (so it contains `name` and other properties if any)? I think we should start being more strict about type checking.
You can add the same prop type here: https://github.com/ryanbaer/busy/blob/9d6899bf4256f224f40a4d610b4914794d14fbaf/src/components/Sidebar/InterestingPeople.js#L32

Reply:
Sure, done.